tunnel_endpoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+AGENT_TUNNEL_ENDPOINT=${AGENT_TUNNEL_ENDPOINT:-http://localhost:4040}
+TAILS_TUNNEL_ENDPOINT=${TAILS_TUNNEL_ENDPOINT:-http://localhost:4040}
+
+WAIT_INTERVAL=${WAIT_INTERVAL:-3}
+WAIT_ATTEMPTS=${WAIT_ATTEMPTS:-10}
+
+retrieve_endpoint () {
+        for _ in $(seq 1 "$WAIT_ATTEMPTS"); do
+                #TODO If wait attempts exceeded and we still haven't successfully gotten an endpoint, exit with status 1
+                if ! curl -s -o /dev/null -w '%{http_code}' "${1}/url" | grep "200" > /dev/null; then
```

Comment:
```suggestion
                if ! curl -s -o /dev/null -w '%{http_code}' "${1}" | grep "200" > /dev/null; then
```
I think we need to remove the `/url` so the function args can specify the endpoint fully and not have anything appended to it.